src/sdc11073/pysoap/soapclientpool.py

```diff
@@ -36,10 +42,12 @@ def register_netloc_user(self, netloc: str, user_ref: UserRef) -> None:
             entry.user_refs.append(user_ref)
 
     def get_soap_client(self, netloc: str, accepted_encodings: list[str], user_ref: UserRef) -> Any:
```

Comment:
return type should be `SoapClientProtocol`

Reply:
done